lib/Target/SubstraitPB/ProtobufUtils.cpp

```diff
@@ -27,12 +27,10 @@ namespace mlir::substrait::protobuf_utils {
 
 namespace pb = ::google::protobuf;
 
-namespace {
 template <typename RelType>
 static const RelCommon *getCommon(const RelType &rel) {
```

Comment:
*If* we decide to keep the anonymous namespaces, we should probably remove the `static` keyword here...